src/obr/core/core.py

```diff
@@ -346,3 +341,84 @@ def map_view_folder_to_job_id(view_folder: str) -> dict[str, str]:
                 if path.absolute().is_relative_to(base.absolute()):
                     ret[job_id] = str(path.absolute().relative_to(base.absolute()))
     return ret
+
+
+def link_folder_to_copy(source: Path) -> Path:
+    """Given a path this functions converts all symlinked files into copies
+    This file does not delete the created .bck folder, neither does it recurse
+    into subfolders.
+
+    Returns: path of the backup folder
+    -------
+    """
+    import os
+
+    source_bck = Path(str(source) + ".bck")
```

Comment:
I assume the `source` path ends with "/"?

Reply:
No it just adds .bck to the folder name ie case/test -> cast/test.bck